<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
  <head>
    <title>PGS-UKBB (2)</title>
    <meta charset="utf-8" />
    <link href="libs/remark-css/default.css" rel="stylesheet" />
    <link href="libs/remark-css/default-fonts.css" rel="stylesheet" />
  </head>
  <body>
    <textarea id="source">




class: title-slide center middle inverse

# Phenome-wide polygenic scores&lt;br&gt;from the UK Biobank

## (2nd meeting)

&lt;br&gt;

### Florian Privé

---

class: center middle inverse

# Introduction

---

### Summary statistics vs individual-level data

&lt;br&gt;

- Summary statistics are not the only way to derive PGS

- Individual-level data can be used directly (i.e. the genotype matrix and the phenotypes)

- This makes sense now with large datasets such as the UKBB

- Plus, we can derive PGS for multiple phenotypes

- Using our fast implem of penalized regressions [1], we can  derive PGS for **240 traits** using **only the UK Biobank** genetic and phenotypic data.

.footnote[[1] Privé, Florian, et al. "Efficient implementation of penalized regression for genetic risk prediction." Genetics 212.1 (2019): 65-74.]

---

### Major strength of the UK Biobank and this study

&lt;br&gt;

- A major concern about PGS is their transferability to other ancestries    
(e.g. bad performances in Africans for PGS trained in Europeans)

- We are well positioned to reiterate this concern with strong evidence

    - UKBB data contains 450K Europeans, 9K South Asians, 2K East Asians and 7K Africans
    
    - who had their genetic and phenotypic information derived **in the same way**
    
    - plus we (I?) expect that LASSO penalty strongly limits the overfitting in the training data, and potentially transfer better to other ancestries

---

class: center middle inverse

# Methods

---

### Genetic data


- 1,040,096 HapMap3 variants 

- Training based on 444,464 individuals of British ancestry +    
testing in 4299 from Poland, 6955 from Italy, 1220 from Iran, 6667 from India, 1846 from China, 2652 from the Caribbean, and 4072 from Nigeria

&lt;img src="../figures/UKBB-matched-country.png" width="62%" style="display: block; margin: auto;" /&gt;

.footnote[Privé, Florian. "Ancestry inference and grouping from principal component analysis of genetic data." bioRxiv (2020).]

---

### Phenotypic data

&lt;br&gt;

- Mapped ICD10 and ICD9 codes to phecodes `\(\rightarrow\)` 104 phecodes with sufficient prediction

- Used other UKBB fields to define 111 continuous and 25 binary phenotypes

    `\(\Rightarrow\)` 111 continuous + 129 binary = **240 phenotypes**
    
&lt;br&gt;

Some examples:

- body measures: height, BMI, BMD, etc.

- blood biochemistry: cholesterol, vitamin D, etc.

- ECG measures

- misc

---

### Prediction accuracy

&lt;br&gt;

**Partial correlation** between PGS (x) and phenotype (y), adjusting for covariates (z).

Basically, regress out x and y by z to get x_eps and y_eps, and compute cor(x_eps, y_eps).

---

class: center middle inverse

# Results

---

### Lasso results in different ancestries

&lt;img src="../figures/lasso-ancestry.JPG" width="80%" style="display: block; margin: auto;" /&gt;

.footnote[Percentage in title = squared slope (in blue)]

---
### Drop in prediction versus distance

&lt;img src="../figures/ratio-vs-dist.JPG" width="80%" style="display: block; margin: auto;" /&gt;

Recall: `\(\text{dist}_{PC}^2 \propto F_{ST}\)`

---

### Manhattan plots for bilirubin and "less tanned"

&lt;img src="../figures/manhattan-bilirubin.JPG" width="80%" style="display: block; margin: auto;" /&gt;&lt;img src="../figures/manhattan-less-tanned.PNG" width="80%" style="display: block; margin: auto;" /&gt;

---

### Restricting to MAF &gt; 5%

&lt;img src="../figures/predict-height-common.PNG" width="90%" style="display: block; margin: auto;" /&gt;

---

### Using family history (LT-FH) to increase power (for GWAS)

&lt;img src="../figures/power-LTFH.JPG" width="70%" style="display: block; margin: auto;" /&gt;

.footnote[Hujoel, M.L.A., Gazal, S., Loh, P. et al. Liability threshold modeling of case–control status and family history of disease increases association power. Nat Genet 52, 541–547 (2020).]

---

### Using LT-FH to increase power (for prediction)

&lt;img src="../figures/lasso-LTFH.JPG" width="100%" style="display: block; margin: auto;" /&gt;

---

class: center middle inverse

# That's all folks!
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
